source/core/ui/view/map/OpenLayerView.js

```diff
@@ -56,9 +56,6 @@ class OpenLayerView extends MapView {
      * @param {Number} [properties.maxZoom=19] - the max zoom value
      * @param {Boolean} [properties.autoZoomOnFirstMarker=false] - auto zoom on the first added marker
      * @param {Object} [properties.initialView] - The initial View can be passed to override the default [View]{@link https://openlayers.org/en/latest/apidoc/module-ol_View-View.html}
-     * @param {Object} properties.initialView.lon - the corresponding longitude in EPSG:4326
-     * @param {Object} properties.initialView.lat - the corresponding latitude in EPSG:4326
-     * @param {Object} properties.initialView.zoom - the default level zoom
      * @param {Object[]} [properties.overlayLayers] - OpenLayers objects to use as overlay layer
      * @param {Object[]} [properties.baseLayers] - OpenLayers objects to use as base layer
      *
```

Comment:
Should the constructor not include an option for projection.  That is, do we need to allow construction passing
```properties.projection``` that can then be read in ```initMap``` routine to set the projection?

Reply:
As per conversation with mdhsl, this is a non-issue